app/config.js

```diff
@@ -1,95 +1,27 @@
-const {homedir} = require('os');
-const {statSync, renameSync, readFileSync, writeFileSync} = require('fs');
-const {resolve} = require('path');
-const vm = require('vm');
-
-const {dialog} = require('electron');
-const isDev = require('electron-is-dev');
 const gaze = require('gaze');
-const Config = require('electron-config');
 const notify = require('./notify');
-
-// local storage
-const winCfg = new Config({
-  defaults: {
-    windowPosition: [50, 50],
-    windowSize: [540, 380]
-  }
-});
-
-let configDir = homedir();
-if (isDev) {
-  // if a local config file exists, use it
-  try {
-    const devDir = resolve(__dirname, '..');
-    const devConfig = resolve(devDir, '.hyper.js');
-    statSync(devConfig);
-    configDir = devDir;
-    console.log('using config file:', devConfig);
-  } catch (err) {
-    // ignore
-  }
-}
-
-const path = resolve(configDir, '.hyper.js');
-const pathLegacy = resolve(configDir, '.hyperterm.js');
+const _import = require('./config/import');
+const _paths = require('./config/paths');
+const win = require('./config/windows');
 
 const watchers = [];
-
 let cfg = {};
 
-function watch() {
-  gaze(path, function (err) {
+const _watch = function () {
+  gaze(_paths.preferencesPath, function (err) {
     if (err) {
       throw err;
     }
     this.on('changed', () => {
-      try {
-        if (exec(readFileSync(path, 'utf8'))) {
-          notify('Hyper configuration reloaded!');
-          watchers.forEach(fn => fn());
-        }
-      } catch (err) {
-        dialog.showMessageBox({
-          message: `An error occurred loading your configuration (${path}): ${err.message}`,
-          buttons: ['Ok']
-        });
```

Comment:
As I thought: 
What if instead of copying default config to a file. We use them and overide them only if the user updated is inside his local config.js file?
It will be easier and if error are found we can still lunch hyper and display better notification inside the shell.